config.js

```diff
@@ -53,10 +53,6 @@ module.exports = {
       'restify': path.join(__dirname, 'src/plugins/plugin-restify.js')
     },
 
-    // Valid entries are:
-    // 'express', 'hapi', 'http', 'restify'
-    excludedHooks: [],
```

Comment:
I guess you could argue that the API change preceded this commit, but the question still stands.